### ISSUE TITLE
Optional Screen Security

### ISSUE THREAD

**Issue by Xashyar** [UnigramDev/Unigram#881]
It's not possible to report errors, and I've found no option under Privacy & Security. the app just blocks screenshots.

**Comment by MarcAnt01** [comment]
I haven't got the aim of this issue, are you reporting something to screen security or about the lack of a button to contact dev?

**Comment by Xashyar** [comment]
Screen Security mainly, but I've also received errors which I can't provide here because there might have been a private chat in the background.

**Comment by MarcAnt01** [comment]
When creating issues, describe them in separate issues not to create confusion

**Comment by MarcAnt01** [comment]
Anyway, what are you asking? As Fela said, screenshot with secret chats are black

**Comment by Xashyar** [comment]
> 
> 
> Would be a privacy violation

@MarcAnt01 

Dude This shouldn't be enabled by default. Even Signal Does Offer this option In the settings.
this should be up to the user to decide. it sometimes becomes necessary.

**Comment by Xashyar** [comment]
Plus, it should only block the Clients Secret Chats, not the whole app.

**Comment by Xashyar** [comment]
Are u suggesting taking a photo from the screen by a Camera? to Report Error :)

**Comment by MarcAnt01** [comment]
Yep, as it's written in contributing guide lines, Unigram follows official clients guide lines, so things not implemented in Telegram official clients, won't be here

**Comment by Xashyar** [comment]
> Yep, as it's written in contributing guide lines, Unigram follows official clients guide lines, so things not implemented in Telegram official clients, won't be here
@MarcAnt01 
u say/write guide lines, I hear/read Blind Lines.

**Comment by Xashyar** [comment]
@FrayxRulez 
Don't u think it must be up to the users to decide? since it's easily possible to take a photo of the device, this is just a detour.
Secret Chat clients must be notified of Screen Shots isn't that Telegram's Official Policy?
